### PR TITLE
resolve #65 repository name capturing from remote

### LINKED ISSUE #65: Repository name capturing from remote is wrong
I have repository in Gitlab called `git@gitlab.com:org/group/subgroup/project.git`.
When trying to use the plugin with releases configuration I am bad request from Sentry with similar message
```
{"refs":["Invalid repository names: subgroup / project"]}
```
Seems it's because of regular expression here https://github.com/arabold/serverless-sentry-plugin/blob/8c60ea5f9ddde97693b60e993ef05588158b5206/src/index.ts#L325

I am not sure what was the motivation for such capturing group as `([^/]+\/[^/]+?)`, so me just putting `(.+?)` is enough and works... this expression allows only one `/` symbol, though it may be few of them in between. What I need is to extract full repository name like `org/group/subgroup/project`.

Is it a bug? Seems that putting repository name manually into `serverless.yaml` fixes it.

### PR DESCRIPTION
**Pull request by fancywriter**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -322,7 +322,7 @@ export class SentryPlugin implements Plugin {
   async _resolveGitRefs(gitRev: GitRev, release: SentryRelease): Promise<SentryRelease> {
     const origin = await gitRev.origin();
     const commit = await gitRev.long();
-    let repository = /[:/]([^/]+\/[^/]+?)(?:\.git)?$/i.exec(origin)?.[1];
+    let repository = /[:/](.+?)(?:\.git)?$/i.exec(origin)?.[1];
     if (repository && origin.includes("gitlab")) {
```
Comment (arabold) [inline src/index.ts:326]: Did you try if this block is still necessary? I would assume we can remove the code altogether.

Comment (fancywriter) [reply src/index.ts:326]: no, not really, I have just quickly tested that regular expression works for my case...

**Comment by fancywriter** [comment]
Actually, this change alone didn't help me... probably something else need to be changed beside this regular expression. Still seeing "Invalid repository names" from sentry.

**Comment by fancywriter** [comment]
Right, sorry, I was wrong, this code doesn't work for me, it's possible in my case to extract repository name from remote because Sentry-Gitlab integration work that way that repo/org names are different from remote... it uses long names with spaces.

Closing this probably, if I come up with an idea how to automate it properly, I can contribute with other changes.

**Comment by arabold** [comment]
Sorry to hear that 😞 I appreciate you looking into it though.

**Pull request closed without merging**